photon-lib/py/test/photonlibpy_test.py

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
Are you running this test anywhere in CI?

Reply:
we are now!